test/controllers/reviews_controller_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class ReviewsControllerTest < ActionController::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+
+  # test "should get new" do
+  #   get new_product_review
+  #   assert_response :success
+  # end
+  #
+  # test "should get create" do
+  #   get :create
```

Comment:
`create` is done via `post` instead of `get`, Then you have to pass it all the things that would normally be in params in order to get tested.